Update pop_importbids.m

nicalbee opened the pull request:
Added code to check for participant's group membership. If the EEG.group variable is empty, then the code looks in the participants.tsv file, adding it to the EEG.group variable if it's found. This is then included in the Study Info GUI in EEGLAB

arnodelorme commented:
This is a great idea, but the code looks too complex. Also are you sure, it does not already work? STUDY.datasetinfo should have as field all the columns of participants.tsv

nicalbee commented:
Thanks @arnodelorme - I have simplified the code now, but unsure whether another pull will interfere with this one. I definitely wasn't getting any of the group information imported before I implemented this adjustment.

arnodelorme commented:
If you update your branch, then the pull request should be automatically be updated.
